buildkite/src/Constants/DebianVersions.dhall

```diff
@@ -27,14 +27,29 @@ let lowerName = \(debVersion : DebVersion) ->
 
 --- Bullseye and Focal are so similar that they share a toolchain runner
 --- Same with Bookworm and Jammy
-let toolchainRunner = \(debVersion : DebVersion) ->
+let toolchainRunner = \(debVersion : DebVersion) -> \(profile : Profiles.Type) ->
+  let baseRunner = merge {
+     Bookworm = RunInToolchain.runInToolchainBookworm
+     , Bullseye = RunInToolchain.runInToolchainBullseye
+     , Buster = RunInToolchain.runInToolchainBuster
+     , Jammy = RunInToolchain.runInToolchainBookworm
+     , Focal = RunInToolchain.runInToolchainBullseye
+   } debVersion
+  in
+  let caqtiRunner = merge {
+      Bookworm = RunInToolchain.runInToolchainCaqtiBookworm
+      , Bullseye = RunInToolchain.runInToolchainCaqtiBullseye
+      , Buster = RunInToolchain.runInToolchainCaqtiBuster
+      , Jammy = RunInToolchain.runInToolchainCaqtiBookworm
+      , Focal = RunInToolchain.runInToolchainCaqtiBullseye
+    } debVersion
+  in
   merge {
-    Bookworm = RunInToolchain.runInToolchainBookworm
-    , Bullseye = RunInToolchain.runInToolchainBullseye
-    , Buster = RunInToolchain.runInToolchainBuster
-    , Jammy = RunInToolchain.runInToolchainBookworm
-    , Focal = RunInToolchain.runInToolchainBullseye
-  } debVersion
+    Standard = baseRunner
+    , Lightnet = baseRunner
+    , BerkeleyMigration = caqtiRunner
```

Comment:
I praise you for finding a way to provide logic for overriding default toolchain runner. For now this change is good, but if we will abandon berkeley_migration profile and use standard like for other, how then we determine that we need to use caqtiRunner?

I wonder if we just won't have a logic on this level.. Instead i would see a global approach on a job level which handle such situation more explicitly, Toolchain runner cannot be determined purely on profile and debian version. What if we will introduce another dhall constant like ToolchainRunner which will encapsulate entire logic of selecting runner? It can have of course standard toolchain but we could also introduce some override mechanism like utilizing some enum:

let ToolchainSelectionMode = < Default | Override: string>

let toolchainRunner = \(debVersion: DebVersion) -> \(mode:  ToolchainSelectionMode)

...

Then we can move selecting toolchain on job level and add some comment so we won't pollute standard toolchains selection and we could explictly write something like:
```

Pipeline.build 
    (ArtifactPipelines.pipeline 
        [ Artifacts.Type.ArchiveMigration ]
        DebianVersions.DebVersion.Bullseye 
        (ToolchainSelectionMode.Type.Override "gcr.io.....") 
        Profiles.Type.BerkeleyMigration
        PipelineMode.Type.PullRequest
    )
```
What you think about it?

I know we are under time pressure so i don't push and i can help you with the change or we can just note that we will do it in the future like that.